openfisca_core/variables.py

```diff
@@ -56,6 +56,17 @@ def to_column(self, tax_benefit_system):
             except ValueError:
                 raise ValueError(u"Incorrect 'end' attribute format in '{}'. 'YYYY-MM-DD' expected where YYYY, MM and DD are year, month and day. Found: {}".format(self.name, end).encode('utf-8'))
 
+        reference = self.attributes.pop('reference', None)
+        if reference:
+            if isinstance(reference, unicode) or isinstance(reference, str):
+                reference = [reference]
+            elif isinstance(reference, list):
+                pass
+            elif isinstance(reference, tuple):
+                reference = list(reference)
+            else:
+                raise TypeError('The reference of the variable {} is a {} instead of a String or a List of Strings.'.format(self.name, type(reference)))
```

Comment:
We are not checking that our reference is a list of string.

If I write `reference = Household,` for instance, there will be no error raised.